### ISSUE TITLE
Broken TLD

### ISSUE THREAD

**Issue by ipmanlk** [eithan1231/whois-light#2]
**Describe the bug**  
.se TLD is broken. Library fails to resolve information for it. 

```js
{
errno: -3008,
code 'ENOTFOUND'
syscall: 'getaddrinfo'
hostname: 'whois.nic-se.se'
}
```

**To Reproduce**
Try to resolve whois information for any .se domain.

**Module version**  
v1.1.2

**Additional context**
I believe ``whois.nic-se.se`` should be updated to ``whois.iis.se``

**Comment by eithan1231** [comment]
I will crawl the icann website over the next few days and update the list.

thank you for reporting!

**Comment by eithan1231** [comment]
I have published version 1.1.3 which will include a fix for this.

A script was also added which will allow for the whois server list to be automatically updated:

https://github.com/eithan1231/whois-light/blob/master/scripts/refresh.js
